### ISSUE TITLE
XBIM XPLORER not rendering on Windows 11 Pro

### ISSUE THREAD

**Issue by pml1711** [xBimTeam/XbimWindowsUI#188]
Hi there
I have just bought a new laptop with Windows 11 pro and I have downloaded the Xbim Xplorer to view some of my IFC files, but for some unknow reason it is not rendering properly. It does not give me any error, it just doesnt render the IFC.
First I thought it could be my IFC, then I downloaded an IFC from here (https://github.com/buildingSMART/Sample-Test-Files/blob/master/IFC%202x3/Duplex%20Apartment/Duplex_A_20110907.ifc), but it did not work either.
Than I thougt I could be an issue with the GPU, but I dont know for sure.
So I am attaching here a short MP4 video showing the issue and I am also attaching images showing my GPU and CPU reference.

CPU/GPU
![CPU_CPU_REF](https://user-images.githubusercontent.com/77131323/185653505-d02c5064-91a4-44cb-a78a-75e5a52d5fed.png)

CONTROL PANEL with Visual C++ installed
![ControlPanel_visual_c++](https://user-images.githubusercontent.com/77131323/185653512-3058a96e-7fd9-4c48-96ce-3fc77553a4b1.png)

MP4 video showing the issue

https://user-images.githubusercontent.com/77131323/185654220-1f9fea14-b0b6-436e-8974-d0831a504ee3.mp4

IFC
[IFC_Duplex_A_20110907.zip](https://github.com/xBimTeam/XbimWindowsUI/files/9383319/IFC_Duplex_A_20110907.zip)

**Comment by CBenghi** [comment]
Hello,
I don't have access to any windows11 installation at the moment.
I'll review once I get the chance to repro.

**Comment by andyward** [comment]
It's been fine on the Win 11 machines I've used (including under ARM WIndows 11). 

Looking at the video it is rendering _something_, but clearly corrupted (e.g. at 04-16 secs). So unlikely to be a missing or broken dependency.

As this is a new machine/install I'd be looking at making sure your graphics drivers are all up to date; and ensure you've the latest Windows update etc.

I'd also maybe test out some other BIM tools and even 3D software (like Paint 3D) to see if you have a more generic issue.

Failing that, share your Device & Windows specification details (from Settings->System->About)

**Comment by Pedrodeo** [comment]
I confirm I have the same issue on a windows 11 Pro. 
I have the issue when I use the integrated GPU to run Xbim (Intel Iris Xe Graphics), but when I switch to my NVIDIA GPU to run it, it works fine.

**Comment by andyward** [comment]
@Pedrodeo any chance you can supply the output from [dxdiag.exe](https://support.microsoft.com/en-us/windows/open-and-run-dxdiag-exe-dad7792c-2ad5-f6cd-5a37-bf92228dfd85) on the affected machine?

How does it handle other 3D content? i.e. can you open this webgl viewer: https://docs.xbim.net/examples/1_Hello_building.live.html

**Comment by pml1711** [comment]
Hi @andyward 
I have run dxgiag.exe on my machine and I am attaching the results. But it seems fine to me, no apparent error and I am also attaching my device and windows specificationn as you suggest. 
I have installed on my machined Solibri and  usBIM viewer by Acca Software and they both work fine. Please see picture also attached.


 Dxdiag
[DxDiag.txt](https://github.com/xBimTeam/XbimWindowsUI/files/9938425/DxDiag.txt)

Hello Building ok
![image](https://user-images.githubusercontent.com/77131323/199979166-7a744d7b-bd9e-4cd0-a5c6-23bdf43f7414.png)


Usbim/ Solibri /Xbim Xplorer
![image](https://user-images.githubusercontent.com/77131323/199980320-2a115714-2af2-42f9-a1d9-fddaed6fbf8b.png)


About my device
On the DxDiag.txt has information about my device.

**Comment by Pedrodeo** [comment]
After some digging, it seems that the issue is more on Intel side, some people experience the same problem with 3D WPF runing with intel Iris XE graphic. That's my issue and apparently it is the same for you @pml1711.

**Comment by andyward** [comment]
Yes, looks like this issue: https://community.intel.com/t5/Graphics/Graphics-failure-with-WPF-3D-content-and-Iris-XE/td-p/1415196

XbimWindowsUI uses Helix toolkit under the covers. I would guess that if you download and run one of the examples you'll see the same issue: https://github.com/helix-toolkit/helix-toolkit 

It may be worth raising a ticket over there if you repro in their examples, as there may be some workaround?

Definitely sounds like not a direct xbim issue - if there's some workaround/recommendation from helix/Intel come back to us.

**Comment by andyward** [comment]
Worth tracking https://github.com/helix-toolkit/helix-toolkit/issues/1862 for possible resolution

**Comment by andyward** [comment]
Unable to reproduce - likely external driver issue
